bot/exts/moderation/infraction/superstarify.py

```diff
@@ -134,6 +135,9 @@ async def superstarify(
         if await _utils.get_active_infraction(ctx, member, "superstar"):
             return
 
+        # Set the duration to 1 hour if none was provided
```

Comment:
This comment is outdated
```suggestion
        # Set to default duration if none was provided
```